tests/unit/Memory.js

```diff
@@ -81,12 +81,13 @@ define([
 			var filter = new store.Filter();
 			assert.strictEqual(store.filter(filter.eq('prime', true)).fetchSync().length, 3);
 			var count = 0;
-			store.filter(filter.eq('prime', true)).fetch().forEach(function (object) {
+			return store.filter(filter.eq('prime', true)).fetch().forEach(function (object) {
```

Comment:
An easier / lower-impact way to make this test still pass would be to just switch to `fetchSync`, since this is using a `Memory` store.